standard-cluster-services/src/main/java/org/terracotta/server/Server.java

```diff
@@ -183,8 +183,22 @@
    * @return
    */
   Properties getCurrentChannelProperties();
-
+  /**
+   * log messages to the console of the server
+   * @param message message to send
+   * @param sub additional parameters
+   */
   void console(String message, Object...sub);
-
+  /**
+   * log messages to the warning of the server
+   * @param warning message to send
+   * @param event additional parameters
+   */
   void warn(String warning, Object...event);
+  /**
+   * log messages to the auditing of the server
+   * @param message message to send
+   * @param event additional parameters
+   */
+  void audit(String message, Properties additionalIdInfo);
```

Comment:
This interface is painfully overloaded - it has several disjoint responsibilities. While we're here, would it be a good time to split this interface into multiple short, meaningful interfaces?

Reply:
we are simply talking about auditing here.  not validation.

Reply:
Server states, CLI process arguments, logging, performing actions (e.g. dump) etc.

Reply:
Ah.  this interface itself.  No I don't think we want to go there.  There are several implementors and consumers.

Reply:
Okay. Yeah, was talking about `Server` interface, not the new method.